packages/components/radio/README.md

```diff
@@ -0,0 +1,131 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - radio</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-radio--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.radio">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.radio" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.radio?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.radio" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.radio">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.radio" height="20"/>
+		</a>
+</p>
+
```

Comment:
Can you add the `event`, `slot`, `properties` to the readme and then use it in the storybook note?